test/utils/setupTruefi2.ts

```diff
@@ -20,7 +20,7 @@ import {
 import { Wallet } from 'ethers'
```

Comment:
Confirming I diffed against setupTruefi2WithLoanFactory3.ts.

Mostly the same apart from naming and expected changes.